polynote-frontend/polynote/ui/display/display_content.ts

```diff
@@ -39,7 +39,7 @@ export function displayContent(contentType: string, content: string | DocumentFr
         const wrapperEl = div(['vega-result'], [targetEl]);
         const spec = JSON.parse(content);
 
-        if (spec?.width === 'container') {
+        if (spec?.width === 'container' || spec?.height === 'container') {
```

Comment:
I'm not entirely sure when this is called, but I think there's an extra `targetEl.style.display = 'block';` in here (since there's already a `targetEl.style.display = 'block';` outside the `if` clause).

Shall I remove it?